src/v2/Utils/getSaleOrLotTimerInfo.ts

```diff
@@ -0,0 +1,91 @@
+export interface Time {
+  days: string
+  hours: string
+  minutes: string
+  seconds: string
+}
+
+interface TimerInfo {
+  copy: string
+  color: string
+}
+
+// If `extendedBiddingEndAt` is passed in and non-null, the `time`
+// parameter refers to the time left in the extended period.
+export const getSaleOrLotTimerInfo = (
+  time: Time,
+  hasStarted?: boolean,
+  saleHasEnded?: boolean,
+  isSaleInfo: boolean = false,
+  extendedBiddingEndAt?: string | null
+): TimerInfo => {
+  const { days, hours, minutes, seconds } = time
+
+  const parsedDays = parseInt(days, 10)
+  const parsedHours = parseInt(hours, 10)
+  const parsedMinutes = parseInt(minutes, 10)
+  const parsedSeconds = parseInt(seconds, 10)
+
+  let copy = ""
+  let color = "blue100"
+
+  // Sale has not yet started
+  if (!hasStarted) {
+    if (parsedDays < 1) {
+      copy = "Bidding Starts Today"
+      // Entered extended bidding
+    } else if (extendedBiddingEndAt) {
```

Comment:
This should likely be in the `else` section, if we are in the extended bidding period the sale has definitely already started.

Reply:
(Also added a test case for this)